### ISSUE TITLE
Close button looks wrong on Accent Title Bar

### ISSUE THREAD

**Issue by DHowett** [microsoft/terminal#18226]
@carlos-zamora this is probably related to your changes for HC

![image](https://github.com/user-attachments/assets/65af7eb4-85fc-4c12-b3ad-ea0c9ce27757)

**Comment by carlos-zamora** [comment]
@DHowett mind sharing your settings? I'll be sure to fix this as a part of #18306 (if it isn't already fixed there).

**Comment by DHowett** [comment]
A-ha!

It only triggers when the tab background is also set to `terminalBackground`.

```json
    "theme": "Accent Tb",
    "themes": 
    [
        {
            "name": "Accent Tb",
            "tab": { "background": "terminalBackground" },
            "tabRow": { "background": "accent" }
        }
    ]
```
